Career/career.html

```diff
@@ -2,24 +2,68 @@
 <html>
 
 <body>
-    <ul>
-        <li>
-            <img src="../Images/20220321113235643.jpg" alt="Reshitha" height="170px"></img><br>
-            <b>Name : Reshitha</b><br>
-            <b>Grade : 2nd Grade</b>
-        </li>
+    <header>
+        <button id="addButton">Add</button>
+    </header>
+    <ul id="ulId">
+        <div id="liIdOne">
+            <li onclick="clickLiOne()">
+                <img src="../Images/20220321113235643.jpg" alt="Reshitha" height="170px"></img><br>
+                <b>Name : Reshitha</b><br>
+                <b>Grade : 2nd Grade</b>
+            </li>
+            <button id="deleteFieldOne">Delete</button>
+        </div>
+
+        <dialog id="dialogOne">
+            <h1> <img src="../Images/20220321113235643.jpg" alt="Reshitha" height="170px"></img><br>
+                <b>Name : Reshitha</b><br>
+                <b>Grade : 2nd Grade</b>
+                <button onclick="closeDialogOne()">close</button>
+            </h1>
+        </dialog>
         </br>
-        <li>
-            <img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img><br>
-            <b>Name : Skandhana</b><br>
-            <b>Grade : 1st Grade</b>
-        </li>
-        <li>
-            <img src="../Images/20220321113910381.jpg" alt="Kalyani" height="170px"></img><br>
-            <b>Name : Kalyani</b><br>
-            <b>Grade : Kinder Garden</b>
-        </li>
+        <div id="liIdTwo">
+            <li onclick="clickLiTwo()">
+                <img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img><br>
+                <b>Name : Skandhana</b><br>
+                <b>Grade : 1st Grade</b>
+            </li>
+            <button id="deleteFieldTwo">Delete</button>
+        </div>
+        <dialog id="dialogTwo">
+            <h1>
+                <img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img><br>
+                <b>Name : Skandhana</b><br>
+                <b>Grade : 1st Grade</b>
+                <button onclick="closeDialogTwo()">Close</button>
+            </h1>
+        </dialog>
```

Comment:
You need to be smart !! Don't hardcode dialog code like this. You can make it dynamic.